app/src/main/java/com/firebase/uidemo/database/Chat.java

```diff
@@ -0,0 +1,28 @@
+package com.firebase.uidemo.database;
```

Comment:
nit: let's just leave chat where it was.  We don't need to access it outside of that context anyway.